### PR TITLE
Add open/close to header + replace icons

### LINKED ISSUE #19: Start in closed state

### PR DESCRIPTION
**Pull request by teesloane**
This PR

Makes it so that the chat app starts closed. 
I also made it so clicking on the header bar can also open the chat.

closes #19 

## UI CHANGES:

jul-16-2017 15-42-37

## teesloane changes:

I want to become this pupper

stubborn-shiba

### CODE REVIEW

File: src/components/Chat/index.js [hunk 1]
```diff
@@ -51,7 +51,10 @@ class Chat extends Component {
 
     return (
       <section className={`Chat--${theme}`}>
-        <Header toggleChat={() => toggleChat(false)} />
+        <Header
+          toggleChat={() => toggleChat(false)}
+          chatOpen={this.props.chatOpen}
```
Comment (mihok) [inline src/components/Chat/index.js]: What's the difference between toggle chat and chat open?

Comment (teesloane) [inline src/components/Chat/index.js]: toggle chat is a function
chatOpen is the boolean state

**Comment by mihok** [comment]
Damn... I did it backwards :(

**Comment by teesloane** [comment]
did what backwards?

**Comment by mihok** [comment]
Did the other PR first, should have been the other way round, now this one has conflicts :/

File: src/components/Chat/index.js (hunk 1 above)
Comment (broneks) [inline src/components/Chat/index.js]: maybe opt for `isChatOpen`?

Comment (teesloane) [inline src/components/Chat/index.js]: good call. i'll refactor that... eventually ...

**Comment by broneks** [comment]
@teesloane can you resolve the conflicts?

**Comment by teesloane** [comment]
ugh fine lol

**Comment by teesloane** [comment]
i will eventually. 

like


later today or somethin'.

**Comment by broneks** [comment]
👍 Cool. Didn't mean right this minute. No rush. I'd just like to see this PR go through!

**Comment by teesloane** [comment]
should be good to merge i think... did it via github

**Pull request merged**